src/tasks/daily.ts

```diff
@@ -525,12 +525,22 @@ export const DailyTasks: Task[] = [
       get("_questPartyFair") === "unstarted",
     completed: () => get("_questPartyFair") !== "unstarted",
     do: () => nepQuest(),
-    outfit: () =>
-      myInebriety() > inebrietyLimit() &&
-      have($item`Drunkula's wineglass`) &&
-      canEquip($item`Drunkula's wineglass`)
-        ? { offhand: $item`Drunkula's wineglass` }
-        : {},
+    outfit: (): OutfitSpec => {
+      const specs: OutfitSpec = { avoid: $items`` };
+      if (
+        myInebriety() > inebrietyLimit() &&
+        have($item`Drunkula's wineglass`) &&
+        canEquip($item`Drunkula's wineglass`)
+      ) {
+        specs.offhand = $item`Drunkula's wineglass`;
+      }
+
+      if (have($item`June cleaver`) && get("_juneCleaverFightsLeft") <= 0) {
+        specs.avoid?.concat($items`June cleaver`);
```

Comment:
`concat` creates a new array rather than modifying an existing array and as such will not do what you want